default_launch.json

```diff
@@ -8,7 +8,13 @@
     "dimensions": {
       "xMeters": 5000,
       "zMeters": 5000
-    }
+    },
+    "legacy_flags": [
+      {
+        "name": "bridge_qos_max_timeout",
+        "value": "0"
+      }
+    ]
   },
```

Comment:
I see Unreal GDK also has max unacked pings rate set, do we know if we may need that too?

https://github.com/spatialos/UnrealGDKStarterProject/blob/master/spatial/default_launch.json#L19-L22